inc/Engine/Optimization/RUCSS/Frontend/Subscriber.php

```diff
@@ -79,6 +79,8 @@ public function treeshake( string $html ) : string {
 		$is_mobile = $this->is_mobile();
 		$used_css  = $this->used_css->get_used_css( $url, $is_mobile );
 
+		$this->used_css->set_cpcss_enabled( (bool) $this->options->get( 'async_css', 0 ) );
```

Comment:
Why `set_cpcss_enabled`? 
Why not inject Options in `used_css` and use in there directly?

Reply:
Yes I thought about that and really I took this decision to separate concerns so the subscriber is dealing with the DB/options and pass it to the usedCss class, but I'm Ok with both options.

Reply:
Could we imagine refactoring the Controller and Subscriber so that Options_Data is only injected in the Controller instead?

We can think about the Subscriber as the router for hooks to callbacks